pkg/nidhogg/decoder.go

```
@@ -21,6 +21,8 @@ func GetConfig(config string) (HandlerConfig, error) {
 		return HandlerConfig{}, fmt.Errorf("error parsing file: %v", err)
 	}
 
+	handlerConf.BuildSelectors()
```

this function should return an error so that we can propagate the error back to main and exit with a clear error message